build/update.sh

```diff
@@ -67,10 +43,15 @@ fi
 
 
 # Update version
-sed -i 's/\(<code id="version">\)[^<]*\(<\/code>\)/\1uglify-es '"$VERSION"'\2/' index.html
+sed -i 's/\(<code id="version">\)[^<]*\(<\/code>\)/\1uglify-js '"$VERSION"'\2/' index.html
+CDN="//registry.npmmirror.com/uglify-js/$VERSION/files"
+CDN="${CDN//\//\\\/}"
+sed -i 's/\(<script src="\)uglify/\1'"$CDN"'/gI' index.html
+
+rm -rf uglify
```

Comment:
I don't see the point of using a CDN here. We should be able to just copy the `uglify` dir into `public`, right?

Reply:
Certainly. I will remove CDN and use local files ( I am in China, CDN speed will be faster )

Reply:
Thanks. If we want to speed things up, it might help to concat the relevant JS files, but that could be done in a follow-up PR if you'd like.